Automattic/jetpack
# Tools: Add eslint-changed.js and use it in CI

#### Changes proposed in this Pull Request:

We have a lot of existing eslint warnings that we don't want to have
complained about in CI.

Since I couldn't find an "eslint-changed" like phpcs-changed, I made
one.

#### Jetpack product discussion


p1607031642022900-slack-CBG1CP4EN, #17970

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Are the many "Unchanged files with check annotations" Github has been reporting gone?
* Create some changes and try it out.

#### Proposed changelog entry for your changes:


* None needed.
